File: python/prism_sdk/throughput_knowledge_representation_workflow_fabric.py
"""Worldgen P04 AFA-worldgen-P04-F15 product wrapper."""
from .worldgen_knowledge_workflow_support import KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt, manifest as _manifest, schedule as _run
FEATURE_ID="AFA-worldgen-P04-F15"
CONTRACT_VERSION="worldgen-throughput-knowledge-workflow/1.0"
def worldgen_throughput_knowledge_representation_workflow_fabric_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="KnowledgeWorkflowRequest1@1", scale="prospective high-throughput", autonomy_tier="A2")
def schedule_worldgen_throughput_knowledge_representation_workflow(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="prospective high-throughput", require_approval=True, require_federation=False)
__all__=["KnowledgeWorkflowRequest","KnowledgeWorkflowReceipt","worldgen_throughput_knowledge_representation_workflow_fabric_manifest","schedule_worldgen_throughput_knowledge_representation_workflow"]

File: crates/worldgen/src/throughput_knowledge_representation_workflow_fabric.rs
//! Worldgen P04-F15 prospective high-throughput knowledge-representation workflow fabric.
use super::knowledge_workflow_support::{self, KnowledgeWorkflowRequest, KnowledgeWorkflowReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P04-F15"; pub const CONTRACT_VERSION:&str="worldgen-throughput-knowledge-workflow/1.0";
pub fn worldgen_throughput_knowledge_representation_workflow_fabric_manifest()->serde_json::Value{knowledge_workflow_support::manifest(FEATURE_ID,CONTRACT_VERSION,"KnowledgeWorkflowRequest1@1","prospective high-throughput","A2")}
pub fn schedule_worldgen_throughput_knowledge_representation_workflow(request:&KnowledgeWorkflowRequest)->Result<KnowledgeWorkflowReceipt,knowledge_workflow_support::KnowledgeWorkflowError>{knowledge_workflow_support::schedule(request,FEATURE_ID,CONTRACT_VERSION,"prospective high-throughput",true,true)}
pub use knowledge_workflow_support::{KnowledgeWorkflowError,KnowledgeWorkflowReceipt as WorldgenThroughputKnowledgeWorkflowReceipt,KnowledgeWorkflowRequest as WorldgenThroughputKnowledgeWorkflowRequest};
